tests/test_dataset_manifest.py

```diff
@@ -467,6 +495,43 @@ def test_image_text_manifest(self):
             for image in dataset_manifest.images:
                 assert image.labels == image_ann[image.id]
 
+    def test_image_matting_manifest(self):
+        zip_file_path = pathlib.Path(__file__).resolve().parent / 'image_matting_test_data.zip'
+        file_reader = FileReader()
+        img_0_matting = Image.open(file_reader.open(str(zip_file_path)+'@mask/test_1.png'))
+        img_1_matting = Image.open(file_reader.open(str(zip_file_path)+'@mask/test_2.png'))
+
+        dataset_manifest = TestCases.get_manifest(DatasetTypes.IMAGE_MATTING, 2)
+        self.assertIsInstance(dataset_manifest, DatasetManifest)
+        self.assertEqual(len(dataset_manifest.images), 2)
+        self.assertEqual(dataset_manifest.images[0].labels, img_0_matting)
```

Comment:
Given the comment about labels in ImageDataManifest, shouldn't this be a list of mattings?